geps/gep-1494/index.md

```diff
@@ -130,20 +130,383 @@ From @ongy, some additional goals to keep in mind:
 
 ## API
 
-(... details, can point to PR with changes)
+This GEP proposes a two-part solution to this problem:
+
+* We introduce a new HTTPRoute Filter, `ExternalAuth`, that allows the
+  specification of an external source to connect to using Envoy's `ext_auth` protocol.
+* We introduce a new Policy object that can be targeted at either the
```

Comment:
Did you consider the possibility of policy at service level? Authorization can be viewed as a service producer requirement, as well as a Gateway mandatory policy or routing level concern.

Reply:
Not really, no, as the primary requirement here is for this to be added as part of _north-south_ routing, and adding this at the Service level would cross over with #3779 for east-west Authorization, which is a whole other thing.